### ISSUE TITLE
Rule: `no-raw-text` - Identify raw text defined in the script

### ISSUE THREAD

**Issue by AvinashRavula** [intlify/eslint-plugin-vue-i18n#451]
**What rule do you want to change?**
no-raw-text

**Does this change cause the rule to produce more or fewer warnings?**
more

**How will the change be implemented? (New option, new default behavior, etc.)?**
new default behavior

**Please provide some example code that this change will affect:**

```Component.vue
<template>
   <p>
      {{ computedValue }}
   </p>
</template>

<script>
export default {
   props: {
      isChecked: {
         type: Boolean,
         default: true
      }
   }
   computed: {
      computedValue() {
         return this.isChecked ? 'Hello' : 'World';
      }
   }
}
```

**What does the rule currently do for this code?**
Does not detect the `Hello` and `World` strings as raw text.

**What will the rule do after it's changed?**
Detects the `Hello` and `World` texts as raw text.

**Additional context**
N/A
